frontend/src/components/Pages/FrontPage/RobotCards/PressureStatusView.tsx

```diff
@@ -20,9 +20,24 @@ const PressureStatusView = ({ pressure, itemSize }: PressureStatusViewProps): JS
     const barToMillibar = 1000
     let icon_color: string = tokens.colors.interactive.primary__resting.hex
 
+    // Define pressure ranges and their corresponding colors
+    const pressureRanges = [
```

Comment:
Define an interface/type for pressureRange.
`interface PressureRange {
min: number;
max: number;
color: string
}`

Reply:
Alternatively define a color with corresponding min and max values. Then construct the pressureRanges from the colors.

Reply:
Just a draft right now - Will request review again. Agree on the comment.